geode-core/src/main/java/org/apache/geode/internal/net/SocketCreator.java

```diff
@@ -350,7 +350,6 @@ private void initialize() {
       try {
         if (this.sslConfig.isEnabled() && sslContext == null) {
           sslContext = createAndConfigureSSLContext();
-          SSLContext.setDefault(sslContext);
```

Comment:
Does the removal of this affect the ability of other code in the same JVM using SSL?

Reply:
This was overriding any system properties set on the JVM to configure SSL.  There were complaints that colocated user code wasn't able to do SSL communication configured with system properties since we were setting the default SSLContext.  There's no reason for us to do that since we create all sockets with SocketCreator.